xbmc/music/dialogs/GUIDialogMusicInfo.cpp

```diff
@@ -319,6 +319,7 @@ CGUIDialogMusicInfo::CGUIDialogMusicInfo(void)
   m_hasUpdatedUserrating = false;
   m_hasRefreshed = false;
   m_bArtistInfo = false;
+  m_scraperAddInfo = false;
```

Comment:
use default member initializer

Reply:
But will that set all these members as required e.g. make new CFileItemList and clear string?

Reply:
I'm only asking to do it for the one variable you added now and yes this constructor is still called.

Reply:
I don't see how doing this one variable (that was previously overlooked by accident) differently to the rest makes better/clearer code.

Reply:
Ah OK, this got missed from global change to defaut member initializers. I will do all bools